spec/models/user_spec.rb

```diff
@@ -28,7 +28,10 @@ def provider_username(service_name)
   let(:other_user) { create(:user) }
   let(:org) { create(:organization) }
 
-  before { omniauth_mock_providers_payload }
+  before do
+    omniauth_mock_providers_payload
+    allow(SiteConfig).to receive(:authentication_providers).and_return(%w[facebook github twitter])
```

Comment:
Instead of hardcoding these you might prefer using `Authentication::Providers.available` - this way when we add more we don't need to go around and add the new one to the list.

It's probably even better to add 

```ruby
allow(SiteConfig).to receive(:authentication_providers).and_return(Authentication::Providers.available)
``` 

to the various functions `omniauth_mock_providers_payload`, `omniauth_mock_facebook_payload`, `omniauth_mock_github_payload`, `omniauth_mock_twitter_payload` so you don't have to explicit this everywhere now that the default value is gone. 

Though I haven't tried, it might not work :) first suggestion still stands

Reply:
> this way when we add more we don't need to go around and add the new one to the list

You know @rhymes I thought about this, but my brain was too fried to realize that sing `Authentication::Providers.available` was the solution 😅